stdlib/asm/math/u64.masm

```diff
@@ -387,9 +431,8 @@ export.unchecked_div
     adv.u64div          # inject the quotient and the remainder into the advice tape
 
     push.adv.1          # read the quotient from the advice tape and make sure it consists of
-    u32assert           # 32-bit limbs
-    push.adv.1          # TODO: this can be optimized once we have u32assert2 instruction
-    u32assert
+    push.adv.1          # 32-bit limbs
```

Comment:
You should be able to replace `push.adv.1 push.adv.1` with `push.adv.2`.

Reply:
Thanks. Changed here and the rest of the places you suggested.